docs/en/observability/apm/getting-started-apm-server/index.asciidoc

```diff
@@ -0,0 +1,153 @@
+[[apm-getting-started-apm-server]]
+= Get started with APM Server
```

Comment:
The APM Server binary page and Fleet-managed APM Server page are very different.

* The binary page explains how to set up and configure the APM Server binary. At the end, you're linked to the APM agent docs.
* The fleet-managed page, however, gets you all the way to viewing data.
* As the install apm agents content is just an include, we could add that content to both guides so that they leave off in the same place.

Reply:
I also don't know about this page title. This is about more than getting started with APM Server. It's about getting started with APM

Reply:
>The APM Server binary page and Fleet-managed APM Server page are very different.

Is this something we should aim to resolve in this PR or separately?

Reply:
I think we should resolve this in this PR.

Reply:
Is this what you had in mind ([preview](https://observability-docs_bk_4230.docs-preview.app.elstc.co/guide/en/observability/master/_apm_server_binary.html#apm-next-steps) / [diff](https://github.com/elastic/observability-docs/pull/4230/commits/228802c5fe084ee1625dede239376a4447fe61aa#diff-694a8aa0d2050a83da700a6d6658ea488cb812b862f8a60152b6db317496a536))?

Reply:
This is exactly what I had in mind. I like it. Do you?

Reply:
Fine with me!